### DISCUSSION TITLE
M5StickC Plus 2 and GPS case

### DISCUSSION THREAD

**Discussion by gkoh** [gkoh/furble#104] [General]
### 
Thanks to @edcooley we now have a 3D printable case that makes using the GPS unit with the M5StickC much nicer.
![image0](https://github.com/user-attachments/assets/e614b825-de4b-4193-86e3-8d6b6d313dcd)
![image1](https://github.com/user-attachments/assets/b9d7e4d5-0720-495d-a90f-2808dd0ba87c)
![image2](https://github.com/user-attachments/assets/00483337-df94-4bf9-8c8f-7c04e50cdbc0)
![image3](https://github.com/user-attachments/assets/1183053c-42f8-4087-800f-8ccbc904f2dd)

### 3MF
[Furble Case w GPS 3mf.zip](https://github.com/user-attachments/files/16230082/Furble.Case.w.GPS.3mf.zip)

### STL
[Furble Case w GPS STL.zip](https://github.com/user-attachments/files/16230083/Furble.Case.w.GPS.STL.zip)

### Construction
The case requires:
* 5x 3mm x 10mm flat screws
* USB Charging Port connectors
   * https://www.amazon.com/dp/B0CJQV9KP8?psc=1&ref=ppx_yo2ov_dt_b_product_details
   * https://www.amazon.com/gp/product/B091CRLJM2/ref=ppx_yo_dt_b_asin_title_o00_s00?ie=UTF8&th=1

**Comment by hijae** [comment]
Hello.
I designed a system that uses both the 18650C battery module and the GPS module simultaneously.

![IMG_2882](https://github.com/user-attachments/assets/715e0091-4f87-4f85-8837-7e9ceb3a7246)
![IMG_2883](https://github.com/user-attachments/assets/511eb19f-be00-4b5a-9694-f60ca0639984)
<img width="619" height="416" alt="스크린샷 2025-07-20 오전 10 16 59" src="https://github.com/user-attachments/assets/5f3921dd-24d4-4657-a017-06a4540a94ef" />


3MF and STEP
[m5stickc18650cGPS.zip](https://github.com/user-attachments/files/22087290/m5stickc18650cGPS.zip)

The case requires
2 x 3mmx10mm screws
